GliaWidgets/SecureConversations/SecureConversations.WelcomeStyle.swift

```diff
@@ -3,40 +3,125 @@ import UIKit
 
 extension SecureConversations {
     /// Theme of the secure conversations welcome view.
-    public class WelcomeStyle {
+    public struct WelcomeStyle: Equatable {
         /// Style of the view's header (navigation bar area).
         public var header: HeaderStyle
 
         /// Title shown in the header.
-        public var title: String
+        public var headerTitle: String
 
         /// Title shown in the welcome area.
-        public var welcomeTitle: String
+        public var welcomeTitleStyle: TitleStyle
 
         /// Description showm in the welcome area.
-        public var welcomeDescription: String
+        public var welcomeSubtitleStyle: SubtitleStyle
+
+        public var checkMessagesButtonStyle: CheckMessagesButtonStyle
```

Comment:
These vars can already have public documentation

Reply:
Addressed in d13ff2b076b1fb2dd2347ac2679f10961f2a0b7b.